docs/documentation.md

```diff
@@ -9,33 +9,49 @@ All request should have in URL `stationId` param encoded with Base64.
 **Tasks:**
 
 - `GET /tasks` - Get all tasks for the authenticated user.
+    - Request
+
+    ```json
+    {
+      "title": "Updated task2", // optional
+      "description": "This is the Updated task", // optional
+      "priority": "HIGH", // optional
+      "isDone": true, // optional
+      "date": "2022-12-23", // optional
+      "notificationTime": "12:04:32" // optional
+    }
```

Comment:
:question: This params are filters to this query?

Reply:
yes